### PR TITLE
Dev jbeckett 0117 RFI only saves when changed/ indigenous Service integration 

### PR DESCRIPTION
**Pull request by jenbeckett**
# Description

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: backend/src/components/rfiApplication.js [hunk 1]
```diff
@@ -95,51 +100,67 @@ async function updateRFIApplication(req, res) {
     let friApplicationResponse = await patchOperationWithObjectId('ccof_rfipfis', rfipfiid, friApplication);
     friApplicationResponse = new MappableObjectForFront(friApplicationResponse, RFIApplicationMappings);
 
-    // update isComplete status
```
Comment (roblo-cgi) [inline backend/src/components/rfiApplication.js:98]: is this a merge issue.  not sure why this is being removed.

File: backend/src/components/rfiApplication.js [hunk 2]
```diff
@@ -153,18 +174,17 @@ async function createRFIApplication(req, res) {
...
     friApplication['ccof_rfi_pfi_dcs_wi_detail_RFI_PFI_Detail'] = req.body.wageList?.map(el=> new MappableObjectForBack(el,DCSWageIncreaseMappings).data);
     friApplication['ccof_rfi_pfi_other_funding_RFI_PFI'] = req.body.fundingList?.map(el=> new MappableObjectForBack(el,OtherFundingProgramMappings).data);
     friApplication['ccof_ccof_rfipfi_ccof_rfipfiexpenseinfo_rfipfi'] = req.body.expenseList?.map(el=> new MappableObjectForBack(el,ExpenseInformationMappings).data);
+    friApplication['ccof_rfipfi_ccof_rfipfi_IndegenousService'] = req.body.indigenousExpenseList?.map(el=> new MappableObjectForBack(el,IndigenousExpenseMappings).data);
 
 
     friApplication['ccof_ApplicationCCFRI@odata.bind'] = `/ccof_applicationccfris(${req.params.ccfriId})`;
     log.info('createRFIApplication payload:', friApplication);
     const friApplicationGuid = await postOperation('ccof_rfipfis', friApplication);
-    //set a flag in ccof_applicationccfri that an RFI exists for this application
```
Comment (roblo-cgi) [inline backend/src/components/rfiApplication.js:165]: merge issue here as well?

File: frontend/src/components/RFI/RFILanding.vue [hunk 3]
```diff
@@ -1490,19 +1491,73 @@ export default {
...
+          //RFI logic ?
+          // this.setRfiList([{name: 'facilityName', guid: 'ccfriguid'}]);
+          // if (this.rfiList?.length > 0) {
+          //   this.$router.push(PATHS.ccfriRequestMoreInfo + '/' + '2dd4af36-9688-ed11-81ac-000d3a09ce90');
+          // } else {
+          this.$router.push({path : `${PATHS.eceweEligibility}`});
+        }
+      }
     },
-    async previous() {
```
Comment (roblo-cgi) [inline frontend/src/components/RFI/RFILanding.vue:1500]: merge issues

File: backend/src/components/rfiApplication.js (hunk 1 above)
Comment (jenbeckett) [reply backend/src/components/rfiApplication.js:98]: I don't think it's being removed, I think the code is just being shifted down?

Comment (jenbeckett) [reply backend/src/components/rfiApplication.js:98]: Yah I think it was a merge issue - I don't recall taking this out. I've added it back in

File: backend/src/components/rfiApplication.js (hunk 2 above)
Comment (jenbeckett) [reply backend/src/components/rfiApplication.js:165]: this one looks like it has moved to line 103?

File: frontend/src/components/RFI/RFILanding.vue (hunk 3 above)
Comment (jenbeckett) [reply frontend/src/components/RFI/RFILanding.vue:1500]: yup

**Pull request merged**